R/readGWLdata2.R

```diff
@@ -0,0 +1,120 @@
+# Copyright 2015 Province of British Columbia
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+#'
+#' Read in groundwater data from files downloaded from GWL tool
```

Comment:
I'll definitely add the roxygen include. That's cool, I had not seen that
before. Credit would be nice too, thank you for suggesting it.

I'll do it tomorrow morning.

CHeers,
Jay

On Fri, Jun 12, 2015 at 1:25 PM, Andy Teucher <notifications@github.com>
wrote:

> In R/readGWLdata2.R
> <https://github.com/bcgov/bcgroundwater/pull/6#discussion_r32351780>:
>
> > @@ -0,0 +1,120 @@
> > +# Copyright 2015 Province of British Columbia
> > +#
> > +# Licensed under the Apache License, Version 2.0 (the "License");
> > +# you may not use this file except in compliance with the License.
> > +# You may obtain a copy of the License at
> > +#
> > +# http://www.apache.org/licenses/LICENSE-2.0
> > +#
> > +# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
> > +# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +# See the License for the specific language governing permissions and limitations under the License.
> > +#'
> > +#' Read in groundwater data from files downloaded from GWL tool
>
> You could add a line saying you're the author:
>
> '# @author Jay R Brown, \email{yourname@@email.com} (note the double @@)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bcgov/bcgroundwater/pull/6/files#r32351780>.
>



-- 
Jay Brown
Senior Project Manager
Easy Metrics Inc.

Direct: 604.961.2762